CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Releases
 v1.17.0-dev (unreleased)
 --------------------
 
--   No changes yet.
+-   Expose backoff.None in backoff api.
```

Comment:
How about something like,

> Export no-op backoff strategy in api/backoff.

From the public API perspective, users don't know what this backoff.None was that you're exposing here.